angular-frontend/src/app/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  <p>Sub: {{ sub }}</p>
```

Comment:
**[suggestion]**: can you expand on this page to indicate where the user is and why they're here? Just to know "hey you've authenticated, this is who you are"...

Reply:
I like it